src/assets/styles/base.scss

```diff
@@ -299,6 +299,11 @@ p {
   border-bottom: 1px dotted;
 }
 
+.text-decoration-dotted-underline {
+  text-decoration-style: dotted;
+  text-decoration-line: underline;
+}
```

Comment:
I wrote this because `.dotted-underline` above doesn't work on multiple lines.

We should probably change users of the old class to this new class but I'm not going to do that now.